compiler/rustc_ast/src/format.rs

```diff
@@ -50,6 +50,7 @@ pub struct FormatArgs {
     ///
     /// Generally only useful for lints that care about the raw bytes the user wrote.
     pub uncooked_fmt_str: (LitKind, Symbol),
+    pub is_source_literal: bool,
```

Comment:
Can you add a doc comment on this field to explain what it means?

Reply:
Done